Azure/sonic-buildimage
# [device]:Add check exception case when read port present fail

**- What I did**
Add check exception case when read port present fail
**- How I did it**
Modify get_psu_presence().  Move read() and close() to "try: ... except IOError:"  To avoid when read
port present sysfs fail. This exception will let sfputil.py to exit. So "sfputil show eeprom"  can't show other port status. 
**- How to verify it**
1. Let I2C CLK/DAGA short in the board(ex:for port1). Or modify cpld drv and let port1 present sysfs to return fail. Do sfputil show eeprom in shell.
Messgse as below, 
..........
File "/usr/share/sonic/device/x86_64-accton_as9716_32d-r0/plugins/sfputil.py", line 129, in get_presence
    content = val_file.readline().rstrip()
IOError: [Errno 6] No such device or address..
2. Add check exception code to sfputil.py. Do the same test. Message as below,
root@sonic:/home/admin# sfputil show eeprom
Error: unable to access file: [Errno 6] No such device or address
Ethernet0: SFP EEPROM not detected

Ethernet4: SFP EEPROM not detected

Ethernet8: SFP EEPROM detected
        Connector: No separable connector
        Encoding: Unspecified
        Extended Identifier: Power Class 1(1.5W max)
        Extended RateSelect Compliance: QSFP+ Rate Select Version 1
        Identifier: QSFP+ or later
        Length Cable Assembly(m): 0
................
.............

**- Description for the changelog**



**- A picture of a cute animal (not mandatory but encouraged)**
